services/app-api/utils/types/reportContext.ts

```diff
@@ -9,6 +15,10 @@ export interface ReportKeys {
 }
 
 export interface ReportMetadataShape extends ReportKeys {
+  newOrExistingProgram: Choice[];
```

Comment:
I'm seeing this option required in all the new types, but we use this type for MLR and NAAAR and it doesn't exist on those types, right?

Reply:
Great call out. 100% this typing needs to be changed to accommodate the differences between the reports.